opencti-platform/opencti-front/src/private/components/analyses/external_references/ExternalReferenceFileImportViewer.tsx

```diff
@@ -227,7 +248,7 @@ ExternalReferenceFileImportViewerBaseProps
                 onClose={() => handleReset()}
                 fullWidth={true}
               >
-                <DialogTitle>{t_i18n('Launch an import')}</DialogTitle>
+                <DialogTitle>{`${t_i18n('Launch an import')}`}</DialogTitle>
```

Comment:
no need to change this line if the title didn't change, right ?

Reply:
It was just an improvement attempt.
I will remove it.